Client/mods/deathmatch/logic/CClientVehicle.cpp

```diff
@@ -5023,3 +5024,17 @@ void CClientVehicle::ResetWheelScale()
 
     m_bWheelScaleChanged = false;
 }
+
+CVector CClientVehicle::GetEntryPoint(std::uint32_t entryPointIndex)
+{
+    static const uint32_t lookup[4] = {10, 8, 11, 9};
```

Comment:
you use `std::uint32_t` everywhere else so please use it here too

Reply:
This is minor, can be done through a follow up commit

Reply:
Spoiler: It'll never happen